webapp/.gitignore

```diff
@@ -1,2 +1,5 @@
 node_modules
 .npminstall
+.vscode/
```

Comment:
This should be redundant, given that `.gitignore` /should/ recurse to the directory root. Can you confirm?

Reply:
Correct. Removed